### PR TITLE
only tas and ticket owner can see timer

### PR DESCRIPTION
**Pull request by n-young**

### CODE REVIEW

File: src/components/queue/QueueListItem/QueueListItem.tsx [hunk 1]
```diff
@@ -98,7 +98,7 @@ const QueueListItem: FC<QueueListItemProps> = ({queue, ticket, position}) => {
                                                                onClick={handleMarkReturned}>
                             I&apos;m back
                         </Button>}
-                        {(isTA || isTicketOwner) && !isCompleted &&
+                        {(isTA || isTicketOwner) && !isCompleted && (isTA || isTicketOwner) &&
```
Comment (nthnluu) [inline src/components/queue/QueueListItem/QueueListItem.tsx:101]: is this expected behavior from previous sign me up?

Comment (n-young) [reply src/components/queue/QueueListItem/QueueListItem.tsx:101]: could be a good param to have, but a few people asked me for it for student privacy

Comment (n-young) [reply src/components/queue/QueueListItem/QueueListItem.tsx:101]: and yes this is how old smu works

Comment (nthnluu) [reply src/components/queue/QueueListItem/QueueListItem.tsx:101]: sg, just wasn't sure

**Pull request merged**